components/bl-transitions-component/src/index.js

```diff
@@ -14,13 +14,13 @@ const transitionsViews = {
 };
 
 export default function Transitions({ component, eventHandlers, pods }) {
-  const { variants, display, duration, dynamicContent } = component;
+  const { variant, display, duration, dynamicContent } = component;
   const { onEndAnimation } = eventHandlers;
 
   const [isContentLoaded, setIsContentLoaded] = useState(!dynamicContent);
 
   const transitionsContainerPod = pods['transitionsContainer'];
-  const Transitions = transitionsViews[variants];
```

Comment:
transitionsViews - TransitionView or TransitionViewMap
also, I thing correct spelling is `transition views`(without s after transition)

Transitions -> Transition or TransitionComponent

Reply:
You miss that: 

transitionsViews -> TransitionView or TransitionViewMap